app/src/main/java/com/everest/userinfo/ConfirmActivity.kt

```diff
@@ -0,0 +1,30 @@
+package com.everest.userinfo
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.text.HtmlCompat
+import com.everest.userinfo.databinding.ActivityConfirmBinding
+
+
+class ConfirmActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityConfirmBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityConfirmBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        val userName = intent.getStringExtra(USERNAME)
+        val email = intent.getStringExtra(EMAIL)
+        val phoneNumber = intent.getStringExtra(PHONE_NUMBER)
+        val pincode = intent.getStringExtra(PINCODE)
+        val address = intent.getStringExtra(ADDRESS)
+
+
+        val resultText =
+            "Hi <b>$userName</b>, How are you? Are you staying at <b>$address-$pincode</b>. I am not able to contact you on <b>$phoneNumber</b>. Can I email you the details at <b>$email</b>"
```

Comment:
We should use string formatting ( from resources) https://stackoverflow.com/questions/3656371/is-it-possible-to-have-placeholders-in-strings-xml-for-runtime-values